File: code/Java/Chapter1/1-3/ChickenRabbit.java
public class ChickenRabbit {
	public static void main(String[] args) {
		count(35, 94);
	}
	
	public static void count(int head, int foot) {
	    for(int chicken = 0; chicken <= head; chicken++) {
	        int rabbit = head - chicken;
	        if(chicken*2 + rabbit*4 == foot) {
				System.out.println("Chicken: " + chicken + ", Rabbit: " + rabbit);
	        }
	    }
	}
}

File: code/C/Chapter1/1-3/chicken_rabbit.c
#include <stdio.h>

void count(int head, int foot) {
    for (int chicken = 0; chicken <= head; chicken++) {
        int rabbit = head - chicken;
        if (chicken * 2 + rabbit * 4 == foot) {
            printf("Chicken: %d, Rabbit: %d\n", chicken, rabbit);
        }
    }
}

int main() {
    count(35, 94);
    return 0;
}